### ISSUE TITLE
Clarify interaction between GTP and minecart contraptions from Create

### ISSUE THREAD

**Issue by Robotgiggle** [gamma-delta/HexMod#426]
Unlike the other types of Create contraptions, minecart contraptions are not included in the `cannot_teleport.json` list. In the past, this meant that you could GTP them to move large structures (like geodes) from place to place. To actually accomplish this, you needed to target your GTP spell not at the contraption but at the minecart itself. When the minecart teleported, it would carry its passenger (the contraption) with it, bringing the whole assembly to the teleport destination

However, the recently created `sticky_teleporters.json` file, which determines which entities are allowed to bring passengers along when teleported, does not include minecarts. This means that teleporting minecart contraptions is no longer possible, since teleporting one is only possible via the passengers-teleport-too mechanic.

If it's intended that minecart contraptions aren't teleportable, they should probably be added to `cannot_teleport.json`. Their current absence from this list implies that they are valid teleport targets, but the `sticky_teleporters.json` mechanic prevents such a teleport from ever actually working.

If it's *not* intended that minecart contraptions aren't teleportable, then either minecarts should be added to the list of entities that can teleport their passengers, or some sort of exception to the "minecarts don't take passengers when teleported" rule should be implemented for minecart contraptions.

**Comment by object-Object** [comment]
Is this still an issue in the latest 1.20.1 prereleases?

**Comment by Robotgiggle** [comment]
Yes, it's still an issue. After some testing in the latest 1.20 version, I can confirm that:
- The minecart contraption entity (internally just `contraption`) is not in the list of entities immune to Greater Teleport (`cannot_teleport.json`)
- It's nonetheless impossible to GTP minecart contraptions, since attempting to GTP the contraption fails in the same way as trying to GTP any other entity that's currently in a minecart, and attempting to GTP the cart fails since minecarts are not included in the list of entities which can be teleported while ridden (`sticky_teleporters.json`).
